snuba/datasets/processors/errors_processor.py

```diff
@@ -332,7 +332,7 @@ def extract_stacktraces(
         frame_stack_levels = []
         exception_main_thread = None
 
-        check_exception_main_thread = get_config("check_exception_main_thread", 0)
+        check_exception_main_thread = get_config("check_exception_main_thread", 1)
```

Comment:
@MeredithAnya I rolled back the default so I can do an integration test [here](https://github.com/getsentry/sentry/pull/46183).
As of now, there's no way to enable the config for the test pipeline, so this is needed.

Reply:
since this is rolled out you can just remove the config check altogether

Reply:
done